packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -2050,6 +2104,108 @@ describe('Checkout', () => {
                 expect(updateCustomerDetailsSpy).toHaveBeenCalledWith({ mobileNumber: newNumber });
             });
         });
+
+        describe('redirectToPayment ::', () => {
+            beforeEach(() => {
+                jest.useFakeTimers();
+            });
+
+            afterEach(() => {
+                jest.clearAllMocks();
+                jest.clearAllTimers();
+            });
+
+            it('should redirect to the payment page after 1 second', () => {
+                // Arrange
+                const windowLocationSpy = jest.spyOn(window.location, 'assign');
```

Comment:
Could this have been done with this?
https://github.com/justeat/fozzie-components/pull/739/files#diff-78d9e6a8935b90cf33c6b12834013ca9bc4ae399aecaae2d684ec50eff9e40a0R81